web/src/pages/Claim.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+   <Layout>
+    <form novalidate class="claim-form">
+      <h1 class="claim-title">Claim Token</h1>
+
+      <md-field>
+        <label>Token</label>
+        <md-input id="token" name="token" v-model="form.token" readonly />
+      </md-field>
```

Comment:
Could you move this field below the project name and hide it until the token is there?

Reply:
@Ajapai Can you do this?

Reply:
I'll try